src/content/target-groups/community-builders.md

```diff
@@ -7,4 +7,4 @@ enable_contact_form: true
 
 A Trustlines community can thrive with just a handful of people. Get started by simply creating trustlines with a couple of friends. If you want to expand your outreach, there are plenty of resources available.
 
-You can utilize and co-create a [Community Toolkit](https://docs.google.com/document/d/1Nnam5NRBxbVqMJ7GrCcldSoFEtf1urMsZt63jfwS644/edit#), take a leadership role and apply for a grant, or connect with others passionate about People Powered Money.
+You can utilize and co-create a [Community Toolkit](community-toolkit.pdf), take a leadership role and apply for a grant, or connect with others passionate about People Powered Money.
```

Comment:
I could solve this for now only by putting a duplicate .pdf file in the same directory.